src/sagemaker/processing.py

```diff
@@ -346,25 +360,26 @@ def run(
         job_name=None,
         experiment_config=None,
     ):
-        """Run a processing job with Script Mode.
+        """Run a processing job.
 
         Args:
-            code (str): This can be an S3 uri or a local path to either
-                a directory or a file with the user's script to run.
-            inputs ([sagemaker.processing.ProcessingInput]): Input files for the processing
-                job. These must be provided as ProcessingInput objects.
-            outputs ([str or sagemaker.processing.ProcessingOutput]): Outputs for the processing
-                job. These can be specified as either a path string or a ProcessingOutput
-                object.
-            arguments ([str]): A list of string arguments to be passed to a
-                processing job.
+            code (str): This can be an S3 URI or a local path to either
```

Comment:
@ajaykarpur I believe this must be a file (not a directory), now, correct?

Reply:
That's correct (take a look at `_handle_user_code_url`).